shard-operator-parent/shard-operator-v1/src/test/resources/application.properties

```diff
@@ -18,3 +18,4 @@ event-bridge.executor.poll-interval.milliseconds=250
 event-bridge.ingress.poll-interval.milliseconds=250
 event-bridge.executor.deployment.timeout-seconds=60
 event-bridge.ingress.deployment.timeout-seconds=60
+quarkus.arc.test.disable-application-lifecycle-observers=true
```

Comment:
Why this change? It looks like you are changing some default behaviour of quarkus

Reply:
yes, with this property application would not listen for Startup/Shutdown Event in testcases. 
Currently we are having additional handling to prevent calling StartupEvent in testcases but with this change we don't need to add `@UnlessBuildProfile(value = "test")` in source classes.

Reply:
Let's continue here https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/1453#discussion_r1060397988